etc/config/c++.amazon.properties

```
@@ -1931,6 +1931,13 @@ libs.abseil.staticliblink=absl_bad_any_cast_impl:absl_failure_signal_handler:abs
 libs.abseil.versions.trunk.version=trunk
 libs.abseil.versions.trunk.path=/opt/compiler-explorer/libs/abseil
 
+libs.belleviews.name=Belle Views
+libs.belleviews.url=https://github.com/josuttis/belleviews
+libs.belleviews.versions=trunk
+libs.belleviews.description=A library of C++ views that just works for all basic use cases as expected.
+libs.belleviews.versions.trunk.version=trunk
+libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/sources
```

I think the final path is
```suggestion
libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/trunk/sources
```